site/docs/v1/tech/events-webhooks/writing-a-webhook.adoc

```diff
@@ -18,7 +18,8 @@ Additional headers may be added to the request by adding headers to the Webhook
 * <<Configuration>>
 ** <<Application Scoped Events>>
 ** <<Tenant Scoped Events>>
-** <<Example Configuration>>
+** <<Example Configuration Before 1.37.0>>
```

Comment:
I would move the pre 1.37 configuration below the post 1.37 configuration.

Reply:
Done